src/infrastructure/server/routes/oauth2/google.ts

```diff
@@ -84,10 +84,7 @@ class GoogleOauth2Api {
 
 const handler = makeInvoker(GoogleOauth2Api)
 
-export const configureGoogleRoutes = (app: Koa): Koa => {
+export const configureGoogleRoutes = (app: Koa): Koa =>
   app
     .use(router.get('/authorize/google', handler('redirectToLogin')))
```

Comment:
We should change `/authorize/google` to `/authorize` as Google is just an implementation detail in our identity management

Reply:
But this is a breaking change so we have to sync with the front-end!